samples/nrf9160/nrf_cloud_mqtt_multi_service/README.rst

```diff
@@ -214,6 +211,15 @@ If you disable both GNSS and cellular-based location tracking, location tracking
   While this sample does allow its core features to be slowed or completely disabled, in real-world applications, you should carefully consider your data throughput and whether MQTT is an appropriate solution.
   If you want to disable or excessively slow all of these features for a real-world application, other solutions, such as the `nRF Cloud Rest API`_, may be more appropriate.
 
+Customizing GNSS antenna configuration
+======================================
+
+This sample takes advantage of the :ref:`lib_modem_antenna` library, which is enabled by default for builds targeting the `Nordic nRF9160 DK`_ and `Nordic Thingy:91`_ (namely, build targets ``nrf9160dk_nrf9160_ns`` and ``thingy91_nrf9160_ns``).
+
+If you are using a different board or build target, or would like to use a custom or external GNSS antenna, see the :ref:`lib_modem_antenna` library documentation for configuration instructions.
+
+Enable :kconfig:option:`CONFIG_MODEM_ANTENNA_GNSS_EXTERNAL` and disable :kconfig:option:`CONFIG_MODEM_ANTENNA_GNSS_ONBOARD` to use an external antenna.
```

Comment:
The antenna selection is done using `choice` in Kconfig, so enabling `CONFIG_MODEM_ANTENNA_GNSS_EXTERNAL` automatically disables `CONFIG_MODEM_ANTENNA_GNSS_ONBOARD`. So in my opinion this could be simply:
```suggestion
Enable :kconfig:option:`CONFIG_MODEM_ANTENNA_GNSS_EXTERNAL` to use an external antenna.
```